pkg/actors/actors.go

```diff
@@ -1111,16 +1118,16 @@ func ValidateHostEnvironment(mTLSEnabled bool, mode modes.DaprMode, namespace st
 	return nil
 }
 
-func (a *actorsRuntime) stateStore() (internal.TransactionalStateStore, error) {
+func (a *actorsRuntime) stateStore() (string, internal.TransactionalStateStore, error) {
```

Comment:
Is there any reason for this? Get name for runtime looks good.

Reply:
The caller of this method is also in a separate package (pkg/actor/reminders), so this avoids passing a new property to the factory method.

Reply:
OK.